dataverse/orgsvc/C#-NETCore/xMultipleSamples/Settings.cs

```diff
@@ -1,18 +1,26 @@
 ﻿namespace PowerPlatform.Dataverse.CodeSamples
 {
-    public static class Settings {
+    public static class Settings
+    {
 
         /// <summary>
         /// The number of records to create for all samples in this solution.
         /// </summary>
-        public const int NumberOfRecords = 100;
+        public const int NumberOfRecords = 100000;
 
         /// <summary>
         /// The maximum number of records operations to send with 
-        /// ExecuteMultiple, CreateMultiple, and UpdateMultiple.
+        /// ExecuteMultiple, CreateMultiple, and UpdateMultiple for Standard entities.
         /// ExecuteMultiple cannot exceed 1000.
         /// </summary>
-        public const short BatchSize = 1000;
+        public const short StandardBatchSize = 1000;
+
+        /// <summary>
+        /// The maximum number of records operations to send with 
+        /// CreateMultiple, UpdateMultiple and DeleteMultiple
+        /// for Elastic entities cannot exceed 100.
+        /// </summary>
+        public const short ElasticBatchSize = 100;
```

Comment:
I don't remember seeing this mentioned in the docs.

Reply:
This is the recommended batch size for xMultiple, we might have to document this.